### ISSUE TITLE
I get the 'Unable to Launch Neovim' error after I quit.

### ISSUE THREAD

**Issue by Bretley** [onivim/oni#1029]
On OSX, if I open a new file I can edit it just fine, everything works, but after I close it I get the error saying to update neovim

**Comment by Bretley** [comment]
Also might be unrelated but I can't get typescript highlighting in oni

**Comment by bryphe** [comment]
> On OSX, if I open a new file I can edit it just fine, everything works, but after I close it I get the error saying to update neovim

Hmm, I'm not able to reproduce this. Couple questions:
- Are you running the latest master build, or the last released build (0.2.17)?
- How are you opening Oni? From a terminal or from the dock?
- How are you closing Oni? `:q` or another way?

Might be that I'm closing Oni in a different way and not seeing it.

> Also might be unrelated but I can't get typescript highlighting in oni

Ah ya, #988 is tracking that. We're switching to a more robust strategy using TextMate themes. It's actually in-progress in master, and can be enabled via the `experimental.textMateHighlighting.enabled` flag (some more info in the wiki: https://github.com/onivim/oni/wiki/Configuration#textmate-highlighting

**Comment by Bretley** [comment]
@bryphe I'm usually always running latest master but I downloaded the latest release binary and it also has the bug. I disabled my init.vim and it stopped so I'm going to try and narrow it down and figure out where the conflict is

**Comment by Bretley** [comment]
```
call plug#begin()

Plug 'scrooloose/nerdtree'

call plug#end()

autocmd VimEnter * NERDTree
autocmd Vimenter * wincmd p
autocmd bufenter * if (winnr("$") == 1 && exists("b:NERDTree") && b:NERDTree.isTabTree()) | q | endif
```
This is it.

**Comment by bryphe** [comment]
Ah ok, thanks for the info! That's helpful to know that it doesn't repro w/o those `autocmd`s.

I'm hoping once #886 is in there will be less of a need for NERDTree anyway 😉 

One other thing I just realized regarding the syntax highlighting - we moved some of the plugins to submodules, so you'll need to run `git submodule update --init --recursive` - otherwise the typescript vim plugin we use won't be around anymore.
